monitor/reducers/approvalReducer.js

```diff
@@ -0,0 +1,100 @@
+import { Log } from 'decentraland-commons'
+
+import { Approval } from '../../src/Approval'
+import { contractAddresses, eventNames } from '../../src/ethereum'
+import { isDuplicatedConstraintError } from '../../src/database'
+import { APPROVAL_TYPES } from '../../shared/approval'
+
+const log = new Log('approvalReducer')
+
+export async function approvalReducer(event) {
+  const { address } = event
+
+  switch (address) {
+    case contractAddresses.LANDRegistry: {
+      await reduceLANDRegistry(event)
+      break
+    }
+    case contractAddresses.EstateRegistry: {
+      await reduceEstateRegistry(event)
+      break
+    }
+    default:
+      break
+  }
+}
+
+async function reduceLANDRegistry(event) {
+  const { name, address } = event
+
+  switch (name) {
+    case eventNames.ApprovalForAll: {
+      // operator and holder are inverted in the contact
```

Comment:
Within the new contracts upgrade, this comment is out of date. We should remove it.